src/main/java/com/waverleysoftware/model/Panel.java

```diff
@@ -0,0 +1,43 @@
+package com.waverleysoftware.model;
+
+/**
+ * A simple model class for panel processing.
+ */
+public class Panel {
```

Comment:
use Lombok to remove boilerplate for all entities

Reply:
I did it in the  **5-lombok-assert-annotation** branch. Please, look at the changes.